### ISSUE TITLE
[med] Chrome + Jaws: Role is not defined for Add Payment.

### ISSUE THREAD

**Issue by Stutikuls** [Expensify/App#8485]
## Action Performed:
1. Using Chrome + Jaws, open URL staging.new.expensify.com
2. Navigate to Setting using Tab key and press Enter to select it.
3. Navigate to payments using Tab key and press using Enter to select it.
4. Navigate to Add payments page using Tab key and press Enter to activate it.
5. Observe the screen reader announcement.

## Expected Result:
Role = "Button" should define for the Add payments control and screen reader should read like "Add Payments button".

## Actual Result:
Screen reader is reading only "Add payments" not reading the role.

## Workaround:
Yes

## Area issue was found in:
Payments Page

## Failed WCAG checkpoints
4.1.2

## User impact:
Screen reader user might be confused without knowing the role of the interactive element, so the user is not able to take any action over the control.

## Suggested resolution:
Add the role="button" for the add payments control or implement the element as a native button, so that the role can be announced by screen reader. Refer to: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

## Platform:
- Web
- Mobile Web

**Version Number:** v1.1.49-1
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Issue reported by:** Stuti 

https://user-images.githubusercontent.com/97600950/161733410-4122fd91-bb0d-4768-95c6-bb97ceade799.mp4
